Automattic/wp-calypso
# Add basic site editor e2e tests

~**NOTE: Tests are failing because template parts aren't loading in the site editor right. Fix is in progress https://github.com/Automattic/wp-calypso/issues/48145**~ Template parts are now loading, ready for review.

#### Changes proposed in this Pull Request

Add e2e tests for site editor. Tests for the following:

* Can navigate to site editor
* Site editor page loads
* Template loads in the site editor
* Template parts load in the site editor

#### Testing instructions

1. Check CI

2. Or you can run the test locally. You will need to decrypt the configuration file. Refer to PCYsg-vnR-p2
```bash
cd test/e2e
NODE_CONFIG_ENV='decrypted' npm run decryptconfig
NODE_CONFIG_ENV='decrypted' ./node_modules/.bin/mocha specs/wp-site-editor-spec.js
```

Fixes #48150
